Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Tabular/Merchant/TransporterConfig.hs

```diff
@@ -56,7 +56,7 @@ mkPersist
       actualRideDistanceDiffThreshold Centesimal
       upwardsRecomputeBuffer Centesimal
       approxRideDistanceDiffThreshold Centesimal
-
+      minAccuracy Int
```

Comment:
why not keep it as `Double` here itself ?

Reply:
Done